### PR TITLE
WIP: 48 & 95 - create public profiles and style for zero or one project

### PR DESCRIPTION
**Pull request by tsyaeger**
-[x] sets currentUserId and isCurrentUser properties on state
-[x] add tabs for filtering
-[x] conditionally render content based on current user status
-[x] refactor display filtering
-[x] adds styling text for empty projects
-[ ] responsive styling 

* the responsive styling is close, but not complete. I could use some assistance on this please.

### CODE REVIEW

File: client/src/components/ProfileDetailPanel.js [hunk 1]
```diff
@@ -1,43 +1,58 @@
...
     [theme.breakpoints.up('md')]: {
-      margin: '30px auto',
+      margin: '25px auto 30px',
+      width: 100,
+      height: 100,
     },
   },
   button: {
-    margin: '10px 0px 10px 0px',
+    width: '80% !important',
```
Comment (evankp) [inline client/src/components/ProfileDetailPanel.js:26]: I haven't needed to use !important in this project, was something preventing you from setting width?

File: client/src/components/ProjectCard.js [hunk 2]
```diff
@@ -18,11 +19,9 @@ const IndustryLabel = styled('div')(({ theme }) => ({
 
 const styles = (theme) => ({
   card: {
-    display: 'flex',
-    flexDirection: 'column',
+    maxWidth: '650px',
```
Comment (evankp) [inline client/src/components/ProjectCard.js]: I didn't have defined widths while restyling on explore page as I let the grid above define the size of card

File: client/src/components/ProjectCard.js [hunk 3]
```diff
@@ -7,6 +7,7 @@ import {
   Typography
 } from '@material-ui/core';
 import PropTypes from 'prop-types';
+const standingProjectPic = 'https://image.freepik.com/free-photo/river-foggy-mountains-landscape_1204-511.jpg'
```
Comment (evankp) [inline client/src/components/ProjectCard.js]: We should change project default picture to the one I made specifically for this purpose in the public images folder

File: client/src/pages/Profile.js [hunk 4]
```diff
@@ -43,63 +62,114 @@ class ProfilePage extends Component {
...
 
-  changeDisplay = (display) => {
-    this.setState({ setDisplay: display })
+  getUserType = () => {
+    if (this.props.match.params.id === undefined){
+      return true
+    } else if (this.props.match.params.id === this.props.userDetails.id){
+      return true
+    }
   }
```
Comment (evankp) [inline client/src/pages/Profile.js:95]: We could probably do this an inline condition, but either way is fine

File: client/src/components/ProfileDetailPanel.js (hunk 1 above)
Comment (tsyaeger) [reply client/src/components/ProfileDetailPanel.js]: Yeah -- it was the only thing that worked for me. Let me know if you have a better solution.

File: client/src/components/ProjectCard.js (hunk 2 above)
Comment (tsyaeger) [reply client/src/components/ProjectCard.js]: Ah -- I just found that it stretched the pics too much. Think that should be done with height?

File: client/src/components/ProjectCard.js (hunk 3 above)
Comment (tsyaeger) [reply client/src/components/ProjectCard.js]: got it. will do

File: client/src/pages/Profile.js (hunk 4 above)
Comment (tsyaeger) [reply client/src/pages/Profile.js]: I wanted the logic to be very explicit

File: client/src/components/ProjectCard.js (hunk 2 above)
Comment (tsyaeger) [reply client/src/components/ProjectCard.js]: just removed it

File: client/src/components/ProjectCard.js (hunk 3 above)
Comment (tsyaeger) [reply client/src/components/ProjectCard.js]: what is the name of the pic? I see ocean, sunset, image not found

**Pull request merged**